package/Dockerfile

```diff
@@ -1,3 +1,4 @@
 FROM alpine:3.4
```

Comment:
We can use `alpine:3.7` here, and skip `amd64=alpine:3.7`

Reply:
Already fixed.Tks